### ISSUE TITLE
PowerApps Cli creating a usersettings.json in the wrong path

### ISSUE THREAD

**Issue by jansonjc** [microsoft/powerplatform-vscode#207]

**Comment by jansonjc** [comment]
@davidjenni that's right. This happens when the pac CLI commands are run within the integrated terminal of vscode. I have a shell script that contains a number of CLI commands (select environment, solution export, unpack, import, etc...) It is quite possible that one of these commands is causing this issue. Since you are able to reproduce this with pcf command, I hope the powerplatform-vscode team will be able to address it. Let me know if you want me to try and figure out exactly which command(s) is causing this.

**Comment by cathalnoonan** [comment]
I'm having the issue as well (macOS). 

It seems like the tilde character returned form [`getAppDataPath()`](https://github.com/microsoft/powerplatform-vscode/blob/main/src/client/telemetry/appdata.ts#L13) isn't resolving to the home directory as expected.

**Comment by mccheesy** [comment]
Any updates on this issue? Can confirm that if I change [src/client/telemetry/appdata.ts:13](https://github.com/microsoft/powerplatform-vscode/blob/cfc45d3f0fe499fb237d4326bf0c1697af35c377/src/client/telemetry/appdata.ts#L13) to `return `${process.env.HOME}/Library/`, I don't get an extra directory structure in the local directory when running PAC commands from VS Code. Not sure why #cathalnoonan marked out their assertion about the `getAppDataPath()` method as that does seem to be the culprit.

**Comment by cathalnoonan** [comment]
I've forgotten the exact reason for crossing that out.

I think the folder was still created when using the `pac` cli directly (without the vscode extension).
